src/utils/api/fieldItem.js

```diff
@@ -11,6 +11,9 @@ export const setItemById = (multiple, item, items) => {
 };
 
 export const getItemsAsArray = (multiple, items) => {
+  if (Array.isArray(items)) {
```

Comment:
Do we care if this is an empty array?

Reply:
I think the fix is a proper fix. As follow up I opened up https://github.com/jsdrupal/drupal-admin-ui/issues/420 to ensure we have unit test coverage for those low level functionality.